frontend/src/components/smart/PvPArenaSummary.vue

```diff
@@ -8,13 +8,22 @@
         <p>
           {{$t('pvp.enterAndWin')}}
           ($SKILL).</p>
-        <div></div>
-        <div class="buttonWrapper">
-          <pvp-button
-            class="pvpButton"
-            @click="handleEnterArenaClick()"
-            :buttonText="$t('pvp.enterArena')"
-          />
+        <div class="buttonsWrapper">
+          <div class="buttonWrapper">
+            <pvp-button
+              class="pvpButton"
+              @click="handleEnterArenaClick"
+              :buttonText="$t('pvp.enterArena')"
+            />
+          </div>
+          <div class="buttonWrapperSecondary">
+            <pvp-button
+              @click="leaveArena"
+              :disabled="loading"
+              :buttonText="$t('pvp.leaveArena')"
+              :secondary="true"
```

Comment:
Just so you know, if you have boolean props you can just add it without anything iirc
so, instead of `:secondary="true"` you should be able to use `secondary`